workspaces/github-actions/packages/backend/src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { Logger } from 'winston';
```

Comment:
Think you should be able to delete this file, don't think it's ever going to be used :pray: